packages/mdc-tab-scroller/_mixins.scss

```diff
@@ -26,10 +26,28 @@
 @import "@material/feature-targeting/mixins";
 @import "./variables";
 
+@mixin mdc-tab-scroller-transition(
+  $duration-ms,
+  $timing-function: $mdc-animation-standard-curve-timing-function,
+  $query: mdc-feature-all()
+) {
+  $feat-animation: mdc-feature-create-target($query, animation);
+
+  .mdc-tab-scroller--animating {
```

Comment:
Add `&` here.

`&.mdc-tab-scroller--animating` 

Mixins are usually applied at root of a component. `.mdc-tab-scroller` in this case.

Reply:
You would also need to make sure that tabscroller accepts custom class names to apply this mixin.

Reply:
Not sure what you mean by custom class names -- is there an example elsewhere I can imitate?

Reply:
Also, I'm a little confused about the &. Currently I've applied this mixin at the root of `@mixin mdc-tab-scroller-core-styles`, *outside* its `.mdc-tab-scroller` rule. If I'm understanding correctly, I should use the & but move the application *inside* `.mdc-tab-scroller { ... }`?

Reply:
The tab scroller template (Google-internal) should accept your class name to apply this mixin.

For example,

```
{call .tabScroller}
  {param classes: css('%CustomTabScroller') /}
{/call}
```

Reply:
Please see my comment here: https://github.com/material-components/material-components-web/pull/5154/files#r332697955

Theme mixins are always applied at root of that component.

Reply:
Oh, that makes sense. Thank you for both clarifications! Done.